metacat-common-server/src/main/java/com/netflix/metacat/common/server/monitoring/LogConstants.java

```diff
@@ -13,95 +13,109 @@
 
 package com.netflix.metacat.common.server.monitoring;
 
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+//CHECKSTYLE:OFF
 /**
  * Log constants.
  */
 public enum LogConstants {
     /**
      * General logging constants.
      */
-    GlobalPrefix("dse"), AppPrefix(GlobalPrefix + ".metacat"),
+    AppPrefix("metacat"),
+
+    /**
+     * evnets.
+     */
+    CounterEventAsync(AppPrefix + ".events.count.Async"),
+    CounterEventSync(AppPrefix + ".events.count.Sync"),
+
+    /**
+     * thrift request.
+     */
+    CounterThrift(AppPrefix + ".thrift.count.request"),
+
+    /**
+     * hive sql lock error.
+     */
+    CounterHiveSqlLockError(AppPrefix + ".count.hiveSqlLockError"),
+    CounterHiveExperimentGetTablePartitionsFailure(AppPrefix + ".hive.count.experimentGetPartitionsFailure"),
+
+    /**
+     * metacat request.
+     */
+    CounterRequestCount(AppPrefix + ".count.request"),
+
+
+    /**
+     * Notifications.
+     */
+    CounterSNSNotificationPartitionAdd(AppPrefix + ".notifications.count.partitionsAdd"),
```

Comment:
I think we should flip it so it's component.measureName.type... I say this cause when we type it into the atlas search filter later on we can better filter down by the metrics and we'll then see that metric name will have a counter, gauge or timer associated with it as opposed to having to back out from the metric to see other metrics cause they're grouped under type... @ajoymajumdar opinion?